### PR TITLE
Allows brand customization

### PR DESCRIPTION
**Pull request by OMaxOe**
## What does this PR do?

- Language files adapted to be able to use variables for domain and app name
- fixed attendeeName on Textfield Placeholder for "meeting_with_user" translation

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Improvement (non-breaking change)

### CODE REVIEW

**Comment by OMaxOe** [comment]
Hey @PeerRich I solved them

**Comment by PeerRich** [comment]
thank you @OMaxOe 

@zomars do we wanna move all these customisations into a DB level with @leog's new "getting started wizard"? or should they stay in .env

File: apps/web/components/App.tsx [hunk 1]
```diff
@@ -259,11 +260,12 @@ const Component = ({
           </ul>
           <hr className="my-6" />
           <small className="leading-1 block text-gray-500">
-            Every app published on the Cal.com App Store is open source and thoroughly tested via peer
-            reviews. Nevertheless, Cal.com, Inc. does not endorse or certify these apps unless they are
-            published by Cal.com. If you encounter inappropriate content or behaviour please report it.
+            Every app published on the {WEBSITE_DOMAIN} App Store is open source and thoroughly tested via
+            peer peer reviews. Nevertheless, {WEBSITE_DOMAIN}, Inc. does not endorse or certify these apps
+            they are published by {WEBSITE_DOMAIN}. If you encounter inappropriate content or behaviour please
+            report it.
```
Comment (PeerRich) [inline apps/web/components/App.tsx:266]: this should stay Cal.com App Store because that is the Cal.com App Store and not the "ACME App Store", no? @zomars @baileypumfleet

File: apps/web/components/ui/PoweredByCal.tsx [hunk 2]
```diff
@@ -17,15 +17,15 @@ const PoweredByCal = () => {
...
             />
           }
           {
             // eslint-disable-next-line @next/next/no-img-element
             <img
               className="relativ -mt-px hidden h-[10px] w-auto dark:inline"
               src="/cal-logo-word-dark.svg"
-              alt="Cal.com Logo"
+              alt={`${WEBSITE_DOMAIN} Logo`}
             />
```
Comment (PeerRich) [inline apps/web/components/ui/PoweredByCal.tsx:29]: this should always stay cal.com

File: apps/web/pages/404.tsx [hunk 3]
```diff
@@ -73,7 +80,7 @@ export default function Custom404() {
                 <ul role="list" className="mt-4">
                   <li className="border-2 border-green-500 px-4 py-2">
                     <a
-                      href="https://console.cal.com"
+                      href={CONSOLE_URL}
                       className="relative flex items-start space-x-4 py-6 rtl:space-x-reverse">
```
Comment (PeerRich) [inline apps/web/pages/404.tsx:84]: this always has to stay console.cal.com, no ned to add CONSOLE_URL to consts

**Review by PeerRich — DISMISSED** [review]

**Comment by PeerRich** [comment]
Danke @OMaxOe, this is a huge PR and we need to discuss some changes internally before merging

**Comment by PeerRich** [comment]
@OMaxOe would you be down to split this PR into two PRs?

- whitelabeling/ brand customisation
- other changes

that way we can merge the "other changes" faster while we discuss the whitelabeling customisations

**Comment by OMaxOe** [comment]
@PeerRich yeah i will seperate it. I will create a pr for "other changes" later

File: apps/web/components/App.tsx (hunk 1 above)
Comment (rene-mueller) [reply apps/web/components/App.tsx:266]: @PeerRich @zomars @baileypumfleet with regard to whiltelabel, this should be possible from my point of view

**Comment by rene-mueller** [comment]
@PeerRich is here anything to do?

File: packages/ui/v2/core/Shell.tsx [hunk 4]
```diff
@@ -711,7 +711,7 @@ function DeploymentInfo() {
         fontSize: "0.5rem",
       }}
       className="mx-3 mt-1 mb-2 hidden opacity-50 lg:block">
-      &copy; {new Date().getFullYear()} Cal.com, Inc. v.{pkg.version + "-"}
+      &copy; {new Date().getFullYear()} {WEBSITE_DOMAIN}, Inc. v.{pkg.version + "-"}
```
Comment (rene-mueller) [inline packages/ui/v2/core/Shell.tsx:714]: here you should used an extra variable like `COMPANY_NAME`.

**Review by servusrene — CHANGES_REQUESTED** [review]

File: apps/web/pages/404.tsx (hunk 3 above)
Comment (rene-mueller) [reply apps/web/pages/404.tsx]: @PeerRich but why is the variable available?

File: apps/web/components/ui/PoweredByCal.tsx (hunk 2 above)
Comment (rene-mueller) [reply apps/web/components/ui/PoweredByCal.tsx]: @PeerRich with regard to whiltelabel, this make no sense...

Comment (zomars) [reply apps/web/components/ui/PoweredByCal.tsx]: Unless we provide some way to get this logo dynamically it will stay as a Cal.com logo and would need code changes to modify anyways

File: packages/emails/templates/team-invite-email.ts [hunk 5]
```diff
@@ -23,10 +25,11 @@ export default class TeamInviteEmail extends BaseEmail {
   protected getNodeMailerPayload(): Record<string, unknown> {
     return {
       to: this.teamInviteEvent.to,
-      from: `Cal.com <${this.getMailerOptions().from}>`,
+      from: `${WEBSITE_DOMAIN} <${this.getMailerOptions().from}>`,
```
Comment (justinforlenza) [inline packages/emails/templates/team-invite-email.ts:28]: I think this (and all emails), should use an App Name environment variable and/or previously suggested company name variable. 

For example I have it hosted at "booking.company.com" I wouldn't want that full domain in the subject of an email.

File: packages/lib/constants.ts [hunk 6]
```diff
@@ -9,6 +9,7 @@ export const WEBSITE_URL = process.env.NEXT_PUBLIC_WEBSITE_URL || "https://cal.c
 export const WEBSITE_DOMAIN = WEBSITE_URL.replace(/http(s):\/\//, "");
 export const APP_NAME = process.env.NEXT_PUBLIC_APP_NAME || "Cal";
 export const SUPPORT_MAIL_ADDRESS = process.env.NEXT_PUBLIC_SUPPORT_MAIL_ADDRESS || "help@cal.com";
+export const COMPANY_NAME = process.env.NEXT_PUBLIC_COMPANY_NAME || "help@cal.com";
```
Comment (rene-mueller) [inline packages/lib/constants.ts:12]: 'Cal.com, Inc.' should be set as default

**Review by servusrene — CHANGES_REQUESTED** [review]

File: packages/lib/constants.ts (hunk 6 above)
Comment (OMaxOe) [reply packages/lib/constants.ts]: thats true

File: packages/emails/templates/team-invite-email.ts (hunk 5 above)
Comment (OMaxOe) [reply packages/emails/templates/team-invite-email.ts:28]: I will change it to app name. Company name could be confusing if it is different from the application.

**Comment by OMaxOe** [comment]
@PeerRich is here antyhing to do ?

File: packages/ui/v2/core/layouts/SettingsLayout.tsx [hunk 7]
```diff
@@ -379,7 +379,6 @@ function ShellHeader() {
             <div className="mb-1 h-6 w-32 animate-pulse rounded-md bg-gray-200" />
           )}
         </div>
-        <div className="ml-auto">{meta.CTA}</div>
```
Comment (zomars) [inline packages/ui/v2/core/layouts/SettingsLayout.tsx:382]: Why was this removed?

File: packages/ui/Kbar.tsx [hunk 8]
```diff
@@ -151,31 +151,31 @@ export const KBarRoot = ({ children }: { children: React.ReactNode }) => {
       section: "Security Settings",
       shortcut: ["t", "f", "a"],
       keywords: "two factor authentication",
-      perform: () => router.push("/settings/security/two-factor-auth"),
+      perform: () => router.push("/settings/security"),
```
Comment (zomars) [inline packages/ui/Kbar.tsx:154]:
```suggestion
      perform: () => router.push("/settings/security/two-factor-auth"),
```

File: packages/ui/Kbar.tsx [hunk 9]
```diff
@@ -151,31 +151,31 @@ export const KBarRoot = ({ children }: { children: React.ReactNode }) => {
...
     },
     {
       id: "impersonation",
       name: "User Impersonation",
       section: "Security Settings",
       shortcut: ["u", "i"],
       keywords: "user impersonation",
-      perform: () => router.push("/settings/security/impersonation"),
+      perform: () => router.push("/settings/security"),
```
Comment (zomars) [inline packages/ui/Kbar.tsx:162]:
```suggestion
      perform: () => router.push("/settings/security/impersonation"),
```

File: packages/ui/Kbar.tsx [hunk 10]
```diff
@@ -151,31 +151,31 @@ export const KBarRoot = ({ children }: { children: React.ReactNode }) => {
...
     },
     {
       id: "webhooks",
       name: "Webhook",
       section: "Developer Settings",
       shortcut: ["w", "h"],
       keywords: "webhook automation",
-      perform: () => router.push("/settings/developer/webhooks"),
+      perform: () => router.push("/settings/developer"),
```
Comment (zomars) [inline packages/ui/Kbar.tsx:170]:
```suggestion
      perform: () => router.push("/settings/developer/webhooks"),
```

File: packages/ui/Kbar.tsx [hunk 11]
```diff
@@ -151,31 +151,31 @@ export const KBarRoot = ({ children }: { children: React.ReactNode }) => {
...
     },
     {
       id: "api-keys",
       name: "API Keys",
       section: "Developer Settings",
       shortcut: ["a", "p", "i"],
       keywords: "api keys",
-      perform: () => router.push("/settings/developer/api-keys"),
+      perform: () => router.push("/settings/developer"),
```
Comment (zomars) [inline packages/ui/Kbar.tsx:178]:
```suggestion
      perform: () => router.push("/settings/developer/api-keys"),
```

**Review by zomars — COMMENTED** [review]
Let's undo the Kbar changes first. Some of the URLs are wrong.

File: packages/ui/Kbar.tsx [hunk 12]
```diff
@@ -252,6 +255,7 @@ const DisplayShortcuts = (item: shortcutArrayType) => {
 
 function RenderResults() {
   const { results } = useMatches();
+  const { t } = useLocale();
```
Comment (zomars) [inline packages/ui/Kbar.tsx:258]:
```suggestion
```

**Review by zomars — APPROVED** [review]
Thank you for your contribution 🙏 @OMaxOe 

Ship it

**Review by servusrene — APPROVED** [review]

File: packages/ui/v2/core/layouts/SettingsLayout.tsx (hunk 7 above)
Comment (OMaxOe) [reply packages/ui/v2/core/layouts/SettingsLayout.tsx]: i can't say why i think it was just a mistake. i have added it back

**Review by justinforlenza — APPROVED** [review]

**Comment by PeerRich** [comment]
@zomars we need to add the new .env to vercel before merging

**Comment by zomars** [comment]
> @zomars we need to add the new .env to vercel before merging

image

It's ok as we have fallback values

**Comment by PeerRich** [comment]
> > @zomars we need to add the new .env to vercel before merging
> 
> image
> 
> It's ok as we have fallback values

oh didn't see that. i added them anyway. cool. we can merge this

**Review by PeerRich — APPROVED** [review]
huge work, thank you!

**Comment by PeerRich** [comment]
@zomars tests failing

**Comment by rene-mueller** [comment]
@PeerRich no Problem. @OMaxOe is a employee from @clicksports

**Comment by OMaxOe** [comment]
@PeerRich thanks. you're welcome :)

**Review by zomars — CHANGES_REQUESTED** [review]
It seems like these changes are breaking [quite a bit of tests](https://github.com/calcom/cal.com/pull/5329/files#diff-7371ade5d8a5edaac577346e5664b3e2e50f9f3fade5887abf9fedf1040c13ff). Blocking to investigate.

**Comment by rene-mueller** [comment]
@zomars are you investigating?

**Comment by zomars** [comment]
> @zomars are you investigating?

This test needs to be updated.

image

**Comment by OMaxOe** [comment]
@zomars i updated the test

**Review by zomars — APPROVED** [review]
Thank you for your contribution 🙏 LGTM

Ship it
